<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpenCVDebugWindow</class>
 <widget class="QWidget" name="OpenCVDebugWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>343</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OpenCV</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPlainTextEdit" name="textOpenCVDebug">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="buttonOpenCVDebugClear">
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonOpenCVStop">
        <property name="text">
         <string>Show OpenCV</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonVideoStop">
        <property name="text">
         <string>Show Service</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonOpenCVDebugClear</sender>
   <signal>clicked()</signal>
   <receiver>textOpenCVDebug</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>196</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>211</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
